libs/cx/src/lib/side-navigation/side-navigation.component.scss

```diff
@@ -1,5 +1,10 @@
 @import '@fundamental-styles/cx/dist/side-nav';
 
+.fdx-side-nav--mobile {
+    width: 100%;
+    border-radius: 0;
+}
+
```

Comment:
is there a plan to move this to fund-styles or it's Angular specific?

Reply:
Already there: https://github.com/SAP/fundamental-styles/pull/4354/files